e2e/support/helpers/e2e-ui-elements-helpers.js

```diff
@@ -219,6 +219,10 @@ export const queryBuilderHeader = () => {
   return cy.findByTestId("qb-header");
 };
 
+export const queryBuilderFiltersPanel = () => {
```

Comment:
:x: Number "between" filter does not work if one of the values is a `number` and the other is a `bigint`


https://github.com/user-attachments/assets/3ec58fdb-8205-438d-af81-98102d940b80

Reply:
:x: Title drill also does not work in that case:

https://github.com/user-attachments/assets/d200ba5d-9d06-47c6-b6bb-e2032d02c1be